terraform/TEMPLATES/calico_nodes.html

```diff
@@ -142,14 +142,7 @@ <h2>K8s Cluster Topology</h2>
       <!-- End Main Content -->
    </div>
 <script src="{{ url_for('static',filename='assets/cui-2.0.0/js/jquery-3.0.0.min.js') }}"></script>
-<script src="{{ url_for('static',filename='assets/cui-2.0.0/js/lodash.min.js') }}"></script>
-<script src="{{ url_for('static',filename='assets/cui-2.0.0/js/clipboard.js') }}"></script>
-<script src="{{ url_for('static',filename='assets/cui-2.0.0/js/styleguide.js') }}"></script>
-<script src="{{ url_for('static',filename='assets/cui-2.0.0/js/highcharts.js') }}"></script>
-<script src="{{ url_for('static',filename='assets/cui-2.0.0/js/exporting.js') }}"></script>
-<script src="{{ url_for('static',filename='assets/cui-2.0.0/js/html2canvas.js') }}"></script>
 <script src="{{ url_for('static',filename='assets/cui-2.0.0/js/functionLibrary.js') }}"></script>
-<script src="{{ url_for('static',filename='assets/js/utils.js') }}"></script>
```

Comment:
Like Shan mentioned the utils.js is actually used. So we shouldn't be removing this one.